src/ProjectTemplates/ProjectTemplatesNoDeps.slnf

```diff
@@ -7,7 +7,8 @@
       "src\\ProjectTemplates\\Web.ItemTemplates\\Microsoft.DotNet.Web.ItemTemplates.csproj",
       "src\\ProjectTemplates\\Web.ProjectTemplates\\Microsoft.DotNet.Web.ProjectTemplates.csproj",
       "src\\submodules\\spa-templates\\src\\Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj",
-      "src\\ProjectTemplates\\test\\ProjectTemplates.Tests.csproj",
+      "src\\ProjectTemplates\\test\\ProjectTemplates.Tests\\ProjectTemplates.Tests.csproj",
+      "src\\ProjectTemplates\\test\\ProjectTemplates.Mvc.Tests\\ProjectTemplates.Mvc.Tests.csproj",
```

Comment:
Same question here about ProjectTemplates.Blazor.Tests.csproj

Reply:
Since we are creating new projects, how about we drop Project prefix from the actual test projects?

```
Templates.Tests.csproj
Templates.Mvc.Tests.csproj
Templates.Blazor.Tests.csproj
```